doc/example-config

```diff
@@ -10,6 +10,7 @@ network:
         macaddress: 00:11:22:33:44:55
       wakeonlan: true
       dhcp4: true
+      mdns: true
```

Comment:
Thank you for providing an example! Could you please additionally add a test case for NetworkManager and a tests case for networkd in `tests/generator/test_common.py` in order to keep the coverage at 100%?